autoload/asyncomplete/utils.vim

```diff
@@ -0,0 +1,11 @@
+" Find a nearest to a `filename` directory `directoryname` by traversing the
+" filesystem upwards
+function! asyncomplete#utils#find_nearest_directory(filename, directoryname) abort
```

Comment:
Can we rename `filename` to `path`. I would like the first parameter to be either `directory` or a `file`.